vitrina/fields.py

```diff
@@ -252,3 +255,41 @@ class DisabledTextInput(TextInput):
 
 class DisabledCharField(CharField):
     widget = DisabledTextInput
+
+
+class StringListWidget(Widget):
+    template_name: str = "component/multi_input.html"
+    validation_errors: list[str | None]
+
+    def value_from_datadict(
+        self, data: QueryDict, files: MultiValueDict, name: str
+    ) -> list[str]:
+        values = data.getlist(name)
+        return [v.strip() for v in values if v.strip()]
```

Comment:
We do not use single letters as variable names, they should be changed:

```suggestion
        return [value.strip() for value in values if value.strip()]
```